### PR TITLE
Enable LLVM 7.0.1 test on Travis

### PR DESCRIPTION
**Pull request by JohanEngelen**

### CODE REVIEW

**Comment by JohanEngelen** [comment]
@kinke The LLVM 7 segfaults, were they spurious or always happening?

**Comment by kinke** [comment]
IIRC, always happening.

File: .travis.yml [hunk 1]
```diff
@@ -31,7 +30,7 @@ matrix:
 
 cache:
   directories:
-    - llvm-7.0.0
+    - llvm-7.0.1
```
Comment (kinke) [inline .travis.yml]: Caching 7.0.0 would still pay off for the OSX job (for which there's apparently no 7.0.1 download yet; IIRC, they don't publish OSX download for minor releases).

Comment (JohanEngelen) [reply .travis.yml]: good catch

**Comment by JohanEngelen** [comment]
> IIRC, always happening.

OK. Build succeeded twice now, so let's merge and see.

**Pull request merged**